### PR TITLE
Feature/tests timeouts

### PR DESCRIPTION
**Pull request by Mingela**
### Description of the Change
Added awkward integration tests timeouts. 

Details: 
https://github.com/junit-team/junit5/issues/80#issuecomment-391313036 
https://blog.philipphauer.de/best-practices-unit-testing-kotlin/

On the one hand it is generally bad idea to use JUnit4 for Kotlin
On the other hand there is no convenient way to implement timeouts in JUnit5 yet

### CODE REVIEW

**Comment by Mingela** [comment]
Let me know if there is no sense in a such wrapping for some tests

File: configs/test_local.properties [hunk 1]
```diff
@@ -25,3 +25,5 @@ test.ethereum.gasLimit=6000000
 # --------- Bitcoin ---------
 test.bitcoin.walletPath=deploy/bitcoin/test-btc.wallet
 test.bitcoin.blockStoragePath=deploy/bitcoin/blocks
+# --------- Timeout (minutes) ---------
+test.testTimeout=5
```
Comment (dolgopolovwork) [inline configs/test_local.properties]: I think we can hardcode this value.

File: notary-integration-test/src/integration-test/kotlin/integration/btc/BtcNotaryIntegrationTest.kt [hunk 2]
```diff
@@ -85,22 +85,24 @@ class BtcNotaryIntegrationTest {
...
-        val btcAmount = 1
-        integrationHelper.sendBtc(btcAddress, btcAmount)
-        Thread.sleep(20_000)
-        val newBalance = integrationHelper.getIrohaAccountBalance(testClient, btcAsset)
-        assertEquals(
-            BigDecimal(initialBalance).add(integrationHelper.btcToSat(btcAmount).toBigDecimal()).toString(),
-            newBalance
-        )
-        assertTrue(btcNotaryInitialization.isWatchedAddress(btcAddress))
+        assertTimeoutPreemptively(timeoutDuration) {
```
Comment (dolgopolovwork) [inline notary-integration-test/src/integration-test/kotlin/integration/btc/BtcNotaryIntegrationTest.kt]: I don't think we need `assertTimeoutPreemptively` in every test. Can we use this construction in tests known to be long-runing only?

File: configs/test_local.properties (hunk 1 above)
Comment (Alexey-N-Chernyshov) [reply configs/test_local.properties]: Agree.

Comment (Mingela) [reply configs/test_local.properties]: Done as a constant value In ConfigHelper

**Comment by Alexey-N-Chernyshov** [comment]
Please, don't use force push for your branches, since it is impossible to view changes from the last review.

**Pull request merged**